with-time.js

```diff
@@ -0,0 +1,57 @@
+const EventEmitter = require('./event-emitter');
+const https = require('https');
+
+class WithTime extends EventEmitter {
```

Comment:
It's working!

But it would be full implementation if you implement WithTime class with this declaration:
```js
class WithTime extends EventEmitter {
   execute(asyncFunc, ...args) {}
} 
```

and it should be invoked this way:
```js
async function fn (...args){
  // .... do the request here
}

const withTime = new WithTime();
withTime.execute(fn)
```

Would you rewrite the implementation?